src/app/left-sidebar/search-doc/search-doc.service.ts

```diff
@@ -75,6 +83,8 @@ export class SearchDocService {
   ): void {
     filter(docs, (doc: HelpDocumentation) => {
       this.populateResults(search, doc.topics, infoMap, results);
+      this.searchContentValue.next(search);
+      this.searchContentValue.complete();
```

Comment:
The complete call is done only for the findFromToc, in case of findFromServer, it is never call. Normal?